apps/aecore/src/aec_governance.erl

```diff
@@ -29,3 +35,24 @@ max_txs_in_block() ->
 
 minimum_tx_fee() ->
     1.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Naming system variables
+
+name_preclaim_expiration() ->
+    300.
+
+name_claim_burned_fee() ->
+    3.
+
+name_claim_max_expiration() ->
+    50000.
+
+name_protection_period() ->
+    2016.
+
+name_claim_preclaim_delta() ->
+    1.
+
+name_registrars() ->
+    [<<"aet">>, <<"test">>].
```

Comment:
From https://github.com/aeternity/protocol/blob/da3a79f9399f73f78833bb74f51ddd66b49ac647/AENS.md#registrars :
> `.aet` will be exclusively available on the mainnet and `.test` will be exclusive to the testnet.

So this shall rather read some kind of reference data (or hardcode different value according to ifdef).

Reply:
@lucafavatella this is incorrect spec. We need to correct it.